src/c-writer.cc

```diff
@@ -322,6 +322,12 @@ class CWriter {
     Allowed,
   };
 
+  bool MaybeWriteNoSandboxFunctionAddress(Offset instruction_offset,
+                                          bool is64bit);
+  bool MaybeWriteNoSandboxMemoryAddress(Offset instruction_offset,
+                                        bool is64bit,
+                                        std::string prefix = "");
```

Comment:
The prefix does not seem to be used at least in this PR

Reply:
Yeah, it will be used in the follow-up.  Do you want me to take this out in the meantime?  (Might not be worth the trouble.)